doc-site/docs/architecture/distributed_sequencer.md

```diff
@@ -0,0 +1,1437 @@
+
+```mermaid
+sequenceDiagram
+  %%{init: { "sequence": { "mirrorActors":false }}}%%
+  participant S as Sender
+  participant C as Coordinator
+  loop Every RetryInterval
+    S -) C: DelegationCommand
+  end
+  alt delegation valid
+    C --) S: DelegationAccepted
+    C -->> C: Coordinate Transaction
+    activate C
+    deactivate C
+  else
+    C --) S: DelegationRejected
+  end
+  
+```
+
+
+##### <a name="message-transaction-delegation-command"></a>Delegation Command
+```proto
+message DelegationCommand {
+    string transaction_id = 1;
+    string delegate_node_id = 2;
+    string delegation_id = 3; //this is used to correlate the acknowledgement back to the delegation. unlike the transport message id / correlation id, this is not unique across retries
+    bytes private_transaction = 4; //json serialized copy of the in-memory private transaction object
+    int64 block_height = 5; // the block height upon which this delegation was calculated (the highest delegation wins when crossing in the post)
+}
+```
+
+As a member of the coordinator committee, a node that receives a `DelegationCommand` must either respond by sending `DelegationAccepted` or `DelegationRejected` message.  
+
+Valid reasons for rejection are
+ - requesters block height is in a different block range than expected.  In this case, the `DelegationRejected` message must contain the block height that the local node is aware of so that the delegating node can make an informed decision based on whether they are behind or ahead.
+ - the current node is not the most preferred available coordinator for the current block height.
+
+If the Delegation is accepted, then the receiving node must begin [coordinating that transaction](#subprocess-coordinate-transaction)
+
+```mermaid
+---
+title: OnDelegationCommand
+---
+flowchart TB
+    A@{ shape: sm-circ, label: "Start" }
+    B@{ shape: diam, label: "Check <br/>block<br/>range" }
+    C@{ shape: lean-r, label: "Reject<br/>(wrong block height)" }
+    D@{ shape: fr-rect, label: "Select<br/>available<br/>coordinator" }
+    E@{ shape: diam, label: "this Is<br/>Coordinator" }
+    I@{ shape: lean-r, label: "Reject<br/>(wrong coordinator)" }
+    G@{ shape: fr-rect, label: "<a href="#subprocess-coordinate-transaction">coordinate transaction</a>" }
+    J@{ shape: fr-circ, label: " " }
+
+
+    A-->B
+    B --> |Behind| C
+    B --> |Ahead| C
+    B --> |Same| D
+    D --> E
+    E --> |Yes| G
+    E --> |No| I
+    G --> J
+```
+
+##### <a name="message-transaction-delegation-rejected"></a>Transaction delegation rejected
+
+The handling of a delegation rejected message depends on the reason for rejection.
+ - if the reason is `MismatchedBlockHeight` and the target delegate is ahead then a new delegation request is sent once the sender has reached a compatible block height.  Compatible block height is defined as a block height in the same block range. 
```

Comment:
Ok - think this is confirmation of my above point - the algorithm above is working to "is compatible block height?" (not "same") semantics.